src/store/user/index.js

```diff
@@ -38,6 +66,37 @@ const mutations = {
 };
 
 const actions = {
+    async loginWithGoogle({ commit, state }) {
+        const gProvider = new GoogleAuthProvider();
+
+        try {
+            const res = await signInWithPopup(auth, gProvider);
+            // const credential = GoogleAuthProvider.credentialFromResult(res);
```

Comment:
remove unwanted code.

Reply:
will remove when final merge happen...may be i need for reference that I tried this earlier didn't work or i have other option to perform same task